topics/reference_guide/custom_language_support/additional_minor_features.md

```diff
@@ -165,6 +165,14 @@ EP: `completion.plainTextSymbol`
 provides a simple way to extract lookup elements from a file so that users have completion available
 in, e.g., plain text editors like VCS commit messages.
 
+### Splitting and Joining List Constructs
+
+EP: `com.intellij.listSplitJoinContext`
+
+[`ListSplitJoinContext`](upsource:///platform/lang-impl/src/com/intellij/openapi/editor/actions/lists/ListSplitJoinContext.kt)
```

Comment:
clarify how to invoke this action in editor/UI

Reply:
Improved drastically